plenum/bls/bls_bft_replica_plenum.py

```diff
@@ -68,6 +86,8 @@ def update_pre_prepare(self, pre_prepare_params, ledger_id):
         pre_prepare_params.append(self._bls_latest_multi_sig.as_list())
         self._bls_latest_multi_sig = None
         # Send signature in COMMITs only
+        if self._all_bls_latest_multi_sigs is not None:
```

Comment:
Can it be that `_bls_latest_multi_sig` is False/None (so that we call `return`), but `_all_bls_latest_multi_sigs ` is not None?